mscore/plugin/api/part.h

```diff
@@ -27,6 +27,7 @@ class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
       Q_PROPERTY(int                            startTrack          READ startTrack)
       Q_PROPERTY(int                            endTrack            READ endTrack)
+      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
```

Comment:
You could append a `// \since 3.2`, so it gets documented as of which version it exists

Reply:
> You could/should append/insert a /// \since MuseScore 3.2 (or /// \since MuseScore 

This seems chicken and the egg... 
How do I know which it will release with before it happens?

I'm not sure how I got the extra commit to. Probably because I thought submitted the pull against the 3.2 branch and it went against master instead. Or a bad cherry-pick on my part.

I assume master is the appropriate location?

I'll fix this right away.

-Dale

Reply:
Just make it `/// \since MuseScore 3.2` for now and change to  `/// \since MuseScore 3.2.1`, should it not get merged into 3.2

Reply:
Suggestions completed.

-Dale